### DISCUSSION TITLE
Welcome to tkvn-site Discussions!

### DISCUSSION THREAD

**Discussion by csmborgaro** [Takagi-VN-Dev/tkvn-site#25] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Xestipular** [comment]
Any update for the development of the game?

**Reply by csmborgaro** [reply]
we're still working on the game, but sadly it's down now to only me now working on it, and I've became busy with projects.
I'm going to get back developing the game once I saved up some funds for it too if given the chance.
